### ISSUE TITLE
specs.json does not take prefix url

### ISSUE THREAD

**Issue by mpandzo** [huge-success/sanic-openapi#37]
If you register the two blueprints with a prefix as follows:

    app.blueprint(openapi_blueprint, url_prefix='/api/openapi')
    app.blueprint(swagger_blueprint, url_prefix='/api/swagger')

swagger ends up not working at /api/swagger because it looks for the specs.json file at /specs.json instead of /api/specs.json

I believe it is because of this code at the bottom of sanic_openapi/openapi.py

```
@blueprint.route('/spec.json')
def spec(request):
    return json(_spec)
```

Should it not take the blueprint prefix into consideration?

**Comment by mpandzo** [comment]
In ui/index.html, 

```
window.onload = function() {
  var url = '/openapi/spec.json';
```

Could be changed to

```
window.onload = function() {
  var url = 'openapi/spec.json';
```
as well

**Comment by DirkGuijt** [comment]
Can we please get this small fix in? That would be great!

**Comment by DirkGuijt** [comment]
I made a change in PR #58 which fixes this and the calls in swagger itself (you have to specify a basePath in the config also according the the OpenAPI 2.0 spec)

**Comment by DirkGuijt** [comment]
this was fixed in #58
